bitcoin/src/taproot/mod.rs

```diff
@@ -429,8 +429,12 @@ impl TaprootBuilder {
         Ok(TaprootBuilder { branch: vec![Some(node)] })
     }
 
-    /// Adds a leaf script at `depth` to the builder with script version `ver`. Errors if the leaves
-    /// are not provided in DFS walk order. The depth of the root node is 0.
+    /// Adds a leaf script at `depth` to the builder with script version `ver`.
+    /// The depth of the root node is 0.
+    ///
+    /// # Errors
+    ///
```

Comment:
In f52afae4fcfdd9ab013cb56bc297c7eef9c73baf:

trailing whitespace here

Reply:
Removed the white space below on line 454